rails/_posts/2016-06-16-build-docker-image.markdown

```diff
@@ -0,0 +1,191 @@
+
+---
+title: Building your Docker Image
+step: 3
+tags:
+- docker
+- rails
+- ruby
+---
```

Comment:
Starting now, you should be writing a very short excerpt.

Example:
`excerpt: Get started with dockerizing your Rails application.`